plugins/node/opentelemetry-instrumentation-knex/src/instrumentation.ts

```diff
@@ -134,24 +165,45 @@ export class KnexInstrumentation extends InstrumentationBase<KnexInstrumentation
           config?.connection?.filename || config?.connection?.database;
         const { maxQueryLength } = instrumentation.getConfig();
 
-        const attributes: api.SpanAttributes = {
+        const attributes: api.Attributes = {
           'knex.version': moduleVersion,
-          [SEMATTRS_DB_SYSTEM]: utils.mapSystem(config.client),
-          [SEMATTRS_DB_SQL_TABLE]: table,
-          [SEMATTRS_DB_OPERATION]: operation,
-          [SEMATTRS_DB_USER]: config?.connection?.user,
-          [SEMATTRS_DB_NAME]: name,
-          [SEMATTRS_NET_PEER_NAME]: config?.connection?.host,
-          [SEMATTRS_NET_PEER_PORT]: config?.connection?.port,
-          [SEMATTRS_NET_TRANSPORT]:
-            config?.connection?.filename === ':memory:' ? 'inproc' : undefined,
         };
+        const transport =
+          config?.connection?.filename === ':memory:' ? 'inproc' : undefined;
+
+        if ((semConv & SemconvStability.OLD) === SemconvStability.OLD) {
```

Comment:
I'm only seeing the case where you have old or stable, but you should also have the case for duplicate that sends both

Reply:
this is defined in the SemconvStability enum that:
```ts
export const enum SemconvStability {
  /** Emit only stable semantic conventions */
  STABLE = 0x1,
  /** Emit only old semantic conventions*/
  OLD = 0x2,
  /** Emit both stable and old semantic conventions*/
  DUPLICATE = 0x1 | 0x2,
}
```
so if it's defined to DUPLICATE the operations `(semConv & SemconvStability.OLD) === SemconvStability.OLD` and `(semConv & SemconvStability.STABLE) === SemconvStability.STABLE` will both return `true` and both conventions will be emitted